src/MLOps.NET/Storage/EntityResolvers/RunResolver.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MLOps.NET.Entities.Impl;
+using MLOps.NET.Storage.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MLOps.NET.Storage.EntityResolvers
+{
+    internal sealed class RunResolver : IEntityResolver<Run>
+    {
+        public Run BuildEntity(IMLOpsDbContext db, Run entity)
```

Comment:
ℹ️ Dev note: I've added these resolvers so we can resuse them if we ever need to load these kind of entities, and they are at one place. This will make it easier to ensure we always correctly populate an entity